Sources/ForageSDK/ForageSDK.swift

```diff
@@ -36,6 +50,8 @@ public class ForageSDK {
             ldManager: LDManager.shared
         )
     }
+    
+    // MARK: - Public API
```

Comment:
Adding these marks gives us this helpful UI menu accessible from the top of Xcode 

https://github.com/user-attachments/assets/b335a4db-ffa4-4765-8b4f-fa7d89e3bc8a